scripts/performance.ps1

```diff
@@ -478,7 +478,7 @@ function Invoke-Test {
     try {
         1..$NumIterations | ForEach-Object {
             Write-LogAndDebug "Running Local: $LocalExe Args: $LocalArguments"
-            $LocalResults = Invoke-LocalExe -Exe $LocalExe -RunArgs $LocalArguments -Timeout $Timeout -OutputDir $OutputDir
+            $LocalResults = Invoke-LocalExe -Exe $LocalExe -RunArgs $LocalArguments -Timeout $Timeout -OutputDir $OutputDir -HistogramFileName "$($Test)_run$($_).txt"
```

Comment:
Is it possible aggregate all raw input and get median instead of median of median of several input files (if I correctly understand this)?

Reply:
No, not with how the histograms are generated (they don't have all the data). We'd instead have to run one big run, instead of 5 smaller ones.